lib/tasks/custom_seed.rake

```diff
@@ -0,0 +1,13 @@
+namespace :db do
+  namespace :seed do
+    Dir[Rails.root.join("db/custom_seeds/*.rb")].each do |filename|
```

Comment:
I prefer the more explicit form, but since `Dir.[]` is an alias for `glob` they are equivalent:

```ruby
Dir.glob(Rails.root.join("db/custom_seeds/*.rb"))
```

Use the one you like better.